info/available-rules.md

```diff
@@ -77,5 +77,6 @@
 |    3 |      3.2 | WRONG_MULTIPLE_MODIFIERS_ORDER          | Check: if multiple modifiers sequence is in the wrong order                                                                                                                  | yes | -                         | - |
 |    3 |      3.14.2 | STRING_TEMPLATE_CURLY_BRACES               | Check: warns if there is redundant curly braces in string template<br> Fix: deletes curly braces                 | yes | - | + |
 |    3 |    3.14.2  | STRING_TEMPLATE_QUOTES                      | Check: warns if there are redundant quotes in string template<br> Fix: deletes quotes and $ symbol                   | yes | - | + |
+|    4 |     4.3.1| NULLABLE_PROPERTY_TYPE                  | Check: if initialize val by null                                                                                                                                             | yes | -                         | - |
```

Comment:
```suggestion
|    4 |     4.3.1| NULLABLE_PROPERTY_TYPE                  | Check: warns if immutable property is initialized with null or if immutable property can have non-nullable type instead of nullable<br>Fix: suggests initial value instead of null or changes immutable property type                    | yes | -                         | - |
```